app/src/main/java/com/sdp/movemeet/Activity/ActivityDescriptionActivity.java

```diff
@@ -53,6 +62,10 @@ protected void onCreate(Bundle savedInstanceState) {
             act = (Activity) intent.getSerializableExtra("activity");
         }
 
+        userId = act.getOrganizerId();
```

Comment:
Could this be messing up the tests because no `Activity` is passed in the `savedInstanceState` bundle during testing?

Reply:
Ok so: in the `MainActivityTest` file, for `mainActivityGoToHome`, it would be best to create a fake activity, pass it to an `Intent`, and then use `ActivityScenario.launch()` to launch the description activity, instead of using `onView().perform().etc`. From there, the test should be able to continue normally.